src/commands/towerverse/cmd.online.ts

```diff
@@ -0,0 +1,41 @@
+// License: GPL-3
+// Maintainer: Otterlord#3653, KittyBorgX#1415
+// Contributors:
+//
+// File description:
+//     The file for the !status command of the TowerBot
+//
+// Extra info:
+//     This file is using the towerverse.js npm package.
```

Comment:
I feel this is a little too specific and not needed. just adds confusion to the project